packages/babel-preset-expo/src/use-dom-directive-plugin.ts

```diff
@@ -8,6 +8,9 @@ import url from 'url';
 
 import { getIsProd } from './common';
 
+const USE_DOM_DIRECTIVE = 'use dom';
+const DOM_COMPONENT_WEBVIEW_ANNOTATION_REGEX = /^\s*(?:\*\s*)?@domComponentWebView\s+(\S+)\s*$/m;
```

Comment:
We should avoid having multiple settings here. There should just be one webview. If we need a setting, it should probably be global or passed to `dom={{ ... }}`

Reply:
let me give `dom={{ ... }}` a try